### PR TITLE
Bug Fixes for ODC 4.7

### PR DESCRIPTION
**Pull request by rishumehra**
This PR has additional Bugfixes for the ODC 4.7.

- Applies to 4.7
- Reviewed by SMEs/QEs
- Aligned team label DevTools

### CODE REVIEW

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1254,6 +1254,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * With Serverless 0.10, the latest supported versions were updated for `trigger`, `subscription`, `channel`, and `IMC`.  Static models corresponding to each showed an API version of `beta`. The API version for eventing resources is now updated to `v1` and the UI now shows the latest supported version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1890104[*BZ#1890104*])
 
+* Previously, in the *YAML view* of the Helm installation form only the YAML schema or code was shown. There was no option to provide and view comments with parameters. Now you can view parameters in defined order with comments. A *Schema* viewer is added to the page, next to *YAML* editor to check all the schema objects present in the Helm Chart. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886861[*BZ#1886861*])
```
Comment (boczkowska) [inline release_notes/ocp-4-7-release-notes.adoc:1257]:
```suggestion
* Previously, in the *YAML view* of the Helm installation form only the YAML schema or code was shown. There was no option to provide and view comments with parameters. Now you can view parameters in a defined order with comments. A *Schema* viewer is added to the page, next to the *YAML* editor to check all the schema objects present in the Helm Chart. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886861[*BZ#1886861*])
```

Comment (Srivaralakshmi) [reply release_notes/ocp-4-7-release-notes.adoc:1257]: LGTM.

Comment (Srivaralakshmi) [reply release_notes/ocp-4-7-release-notes.adoc:1257]: LGTM

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1258,6 +1258,10 @@ If you are using the option `--keep-manifest-list=true`, the only valid value fo
 
 * Previously, monitoring alerts with severity levels such as `critical` and `warning` were treated as `info` level alerts. As a result, the *Monitoring Alert* icon was not displayed on the workload in the *Topology* view for these alerts. This issue is now fixed; alerts like `critical` are treated as `warning` level alerts and a *Monitoring Alert* icon is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1925200[*BZ#1925200*])
 
+* Previously, in the *YAML view* of the Helm installation form only the YAML schema or code was shown. There was no option to provide and view comments with parameters. Now you can view parameters in a defined order with comments. A *Schema* viewer is added to the page, next to the *YAML* editor to check all the schema objects present in the Helm Chart. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1886861[*BZ#1886861*])
```
Comment (rohitkrai03) [inline release_notes/ocp-4-7-release-notes.adoc:1261]: YAML schema was not shown earlier. Only YAML code was shown. This bug fix adds the schema viewer in yaml editor. Also, the comments are not shown in YAML code. Only the schema and its descriptions.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-7-release-notes.adoc (hunk 2 above)
Comment (rishumehra) [reply release_notes/ocp-4-7-release-notes.adoc:1261]: Thanks for the clarification @rohitkrai03. Will update the content based on this comment.

**Comment by rohitkrai03** [comment]
LGTM

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**